types/types.ts

```diff
@@ -0,0 +1,22 @@
+interface SheetsConfig {
+  type: string;
+  options: {
+    spreadsheetId: string;
+    tableName: string;
+    clientSecret: Object;
+  };
+}
+
+interface MetricsResults {
+  timestamps: Array<any>;
```

Comment:
Good point. I was thinking to do that after merging https://github.com/paulirish/pwmetrics/pull/50